cli/cli.go

```diff
@@ -91,7 +91,7 @@ func NewDefraCommand(cfg *config.Config) *cobra.Command {
 	collection := MakeCollectionCommand(cfg)
 	collection.AddCommand(
 		MakeCollectionGetCommand(),
-		MakeCollectionKeysCommand(),
+		MakeCollectionListDocIDsCommand(),
```

Comment:
question: Why did you add `List` into the func name, and file name?

Reply:
Because I was a bit confused what `...CollectionKeysCommand()` actually does. Specially when the others have descriptive actions within their names like `...CollectionUpdateCommand()` and `...CollectionGetCommand()`.

https://github.com/sourcenetwork/defradb/commit/c1eef2081763e4439359a3e88b6191feeba747af#diff-8a4cbfe5ceaae77d6bccca6c6036c192c811898e4392cd40ae7bb8e2e2ce6ae9

From above, it seemed like it actually lists the collection's docIDs hence the name change